shopify/actions/cancel_order/request_url.gtpl

```diff
@@ -0,0 +1,4 @@
+{{- $shop := .integration.configuration.shop}}
+{{- $apiVersion := "2025-01" }}
```

Comment:
it might make sense to update all of these to 2025-04. I think there are no changes needed.